core/app/views/brightcontent/base/index.html.erb

```diff
@@ -19,5 +19,7 @@
 </div>
 
 <div class="panel-footer panel-footer-pagination">
-  <%= will_paginate collection, renderer: BootstrapPagination::Rails %>
+  <%= render partial: "page_sizes",
+    locals: { sizes: per_page_sizes, current_size: items_per_page } %>
```

Comment:
I took another look and I think it's not really necessary here.
Let's call it a personal preference.